README.rst

```diff
@@ -293,20 +293,42 @@ path to configuration file
 .. code-block:: bash
 
     # command line log file
-    mlx-warnings --config pathtoconfig.json junit_output.xml
+    mlx-warnings --config path/to/config.json junit_output.xml
     # command line command execution
-    mlx-warnings --config patchtoconfig.json --command <commandforjunit>
+    mlx-warnings --config path/to/config.json --command <commandforjunit>
 
 
 -------------
 Other options
 -------------
 
-Since plugin is under active development there are new Features added fast.
+Since the plugin is under active development there are new Features added fast.
 Important options currently include setting maximum number of warnings or
 minimum number of warnings, that are still acceptable to return 0 (success)
 return code. Look at scripts help, for more details about the options.
 
+Exclude matches with regex
+--------------------------
+
+In case you want a checker to exclude certain matches, you can configure
+a regular expression in the configuration file on a per-checker basis. (Only
+sphinx, doxygen and xmlrunner checkers are currently supported by this feature.)
+If this regex' pattern is found in a match of the checker's regex, the checker
+won't count that match. Add the regex as a string value for the `exclude` key.
+An example configuration for the sphinx checker is given below:
+
+.. code-block:: json
+
+    {
+        "sphinx":{
+    	    "enabled": true,
+    	    "min": 0,
+    	    "max": 0,
+            "exclude": "RemovedInSphinx\\d+Warning"
+        }
```

Comment:
Alignment here, but because of that can we make it array of strings for exclude regexes?

Reply:
It is possible, but a user can use `|` OR-logic inside this single regex to achieve the same result, right?

Reply:
True, but it is less readable that way since if you want to exclude 5 different error messages you will have a lot of brackets (especially with more complex error parsing).

Reply:
It must now be a list: df6e6440110e075724cf6a44d9b7781b62d27f9b